File: Kattis/bestagjofin/bestagjofin.py
n = int(input())
name, fun_metric = input().split()
fun_metric = int(fun_metric)

best_metric = fun_metric
best_name = name

while n > 1:
    name, fun_metric = input().split()
    fun_metric = int(fun_metric)

    if fun_metric > best_metric:
        best_metric = fun_metric
        best_name = name
    n-= 1
print(best_name)

File: Kattis/bestagjofin/bestagjofin.cpp
#include <iostream>
#include <string>

int main() {
    int n, fun_metric, best_metric;
    std::string name, best_name;

    std::cin >> n >> name >> fun_metric;
    best_name = name;
    best_metric = fun_metric;

    while (n > 0) {
        std::cin >> name >> fun_metric;
        if (fun_metric > best_metric) {
           best_metric = fun_metric;
           best_name = name;
        }
        n--;
    }
    std::cout << best_name << std::endl;
    return 0;
}
